content/docs/tasks/traffic-management/request-routing/index.md

```diff
@@ -39,7 +39,7 @@ to the Bookinfo app. If you already created conflicting virtual services for Boo
 1.  Run the following command to apply the virtual services:
```

Comment:
Remove (just above):
  > Before continuing, be sure you don't have any existing virtual services applied
to the Bookinfo app. If you already created conflicting virtual services for Bookinfo, you must use `replace` rather than `create` in the following command.